.github/workflows/node.js.yml

```diff
@@ -0,0 +1,24 @@
+name: Node.js CI
```

Comment:
Could you add a quick comment up top along the lines of `# This file can be deleted if your project is not using Github workflow`

Reply:
Sure